creditnote.go

```diff
@@ -31,6 +31,15 @@ const (
 	CreditNoteTypePrePayment  CreditNoteType = "pre_payment"
 )
 
+// CreditNoteLineItemType is the list of allowed values for the credit note line item's type.
+type CreditNoteLineItemType string
+
+// List of values that CreditNoteType can take.
+const (
+	CreditNoteLineItemTypeCustomLineItem  CreditNoteLineItemType = "custom_line_item"
+	CreditNoteLineItemTypeInvoiceLineItem CreditNoteLineItemType = "invoice_line_item"
+)
+
 // CreditNoteParams is the set of parameters that can be used when creating or updating a credit note.
 // For more details see https://stripe.com/docs/api/credit_notes/create, https://stripe.com/docs/api/credit_notes/update.
 type CreditNoteParams struct {
```

Comment:
Should this one have `Lines`?  seems undocumented, but also the api-ref shows Lines in the CreditNote create call for dotnet

Reply:
Ack nice catch, yes it should, fixing.